edx_exams/apps/api/v1/tests/test_views.py

```diff
@@ -815,6 +815,26 @@ def test_access_no_due_date(self, attempt_status, response_status):
         if response_status == 200:
             self.assert_valid_exam_access_token(response, self.user, no_due_date_exam)
 
+    def test_access_user_is_course_staff(self):
+        """
+        Verify the endpoint grants access for an exam
```

Comment:
If you agree with my suggestion below, please update the docstring and the test.